orcid-web/src/main/java/org/orcid/frontend/web/controllers/BaseController.java

```diff
@@ -353,7 +358,10 @@ protected void logoutCurrentUser(HttpServletRequest request, HttpServletResponse
 		}
 		if (authentication != null && authentication.isAuthenticated()) {
 			new SecurityContextLogoutHandler().logout(request, response, authentication);
```

Comment:
@wjrsimpson hum I thought this line caused the a new token to be made.? Please verify that wasn't the case.